test/e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,167 @@
+package e2e
```

Comment:
As discussed yesterday @qu1queee this is mandatory within src directories where production and test code resides side by side. e2e lives in its own directory. Changing the package to `e2e_test` would cause validation errors on the helper files like `validators.go`. One would then move all files in another `test` sub-directory. I do not see much value in this. So, let's leave it as it is.